src/App.tsx

```diff
@@ -1,24 +1,154 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { addTodo, deleteTodo, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Errors } from './types/Errors';
+import { SortType } from './types/SortType';
+import { TodoList } from './components/TodoList';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { TodoItem } from './components/TodoItem';
 
-const USER_ID = 0;
+const USER_ID = 121;
+
+function filterTodoList(todos: Todo[], sortKey: SortType) {
+  switch (sortKey) {
+    case SortType.Completed:
+      return todos.filter(todo => todo.completed);
+
+    case SortType.Active:
+      return todos.filter(todo => !todo.completed);
+
+    default:
+      return todos;
+  }
+}
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [inputText, setInputText] = useState('');
+  const [errorMessage, setErrorMessage] = useState<Errors | ''>('');
+  const [sortBy, setSortBy] = useState<SortType>(SortType.All);
+
+  const filteredTodos = filterTodoList(todos, sortBy);
+
+  const COMPLETED_TODO = filterTodoList(todos, SortType.Completed).length;
+  const ACTIVE_TODO = todos.length - COMPLETED_TODO;
```

Comment:
don't use upperCase for internal consts

as a rule upperCase is used for general conts like `USER_ID` which we keep in some file